src/domains/integration/service/integration.service.impl.ts

```diff
@@ -67,7 +69,7 @@ export class IntegrationServiceImpl implements IntegrationService {
             pendingProjectId: pendingProject.id,
           }),
         { delay: 1000 * 60 * 60 * 2, retries: 3 },
-        [ConflictException, PrismaClientKnownRequestError, PrismaClientUnknownRequestError]
```

Comment:
Please add ConflictException again, if it is thrown, integration must not be repeated.